packages/expo-symbols/expo-module.config.json

```
@@ -0,0 +1,9 @@
+{
+  "platforms": ["ios", "android"],
```

```suggestion
  "platforms": ["ios"],
```

do we support android?

---

Yes, we'll be adding support on android

---

cool good to know!